glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt

```diff
@@ -260,45 +231,9 @@ class LabeledMetricTypeTest {
         )
     }
 
-    @Ignore("Timespan metric not yet implemented")
-    @Test
-    fun `Test labeled timespan metric type`() {
-        /*
-        TimespansStorageEngine.clearAllStores()
-
-        val timespanMetric = TimespanMetricType(
-            disabled = false,
-            category = "telemetry",
-            lifetime = Lifetime.Application,
-            name = "labeled_timespan_metric",
-            sendInPings = listOf("metrics"),
-            timeUnit = TimeUnit.Nanosecond
-        )
-
-        val labeledTimespanMetric = LabeledMetricType<TimespanMetricType>(
-            disabled = false,
-            category = "telemetry",
-            lifetime = Lifetime.Application,
-            name = "labeled_timespan_metric",
-            sendInPings = listOf("metrics"),
-            subMetric = timespanMetric
-        )
-
-        labeledTimespanMetric["label1"].start(this)
-        labeledTimespanMetric["label1"].stopAndSum(this)
-        labeledTimespanMetric["label2"].start(this)
-        labeledTimespanMetric["label2"].stopAndSum(this)
-
-        assertTrue(labeledTimespanMetric["label1"].testHasValue())
-
-        collectAndCheckPingSchema(Pings.metrics).getJSONObject("metrics")!!
-        */
-    }
-
     @Ignore("UUID metric not yet implemented")
```

Comment:
AAARG, this fell through the cracks. Any chance you could file a bug about this? We should either remove it from a-c or implement it here.

Reply:
Hm, I actually thought we _had_ an issue, that's why I left it in.
Labeled UUIDs and labeled string lists are not documented on our docs either, so I considered them "not implemented"

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1570303